selfswab/models.py

```diff
@@ -56,6 +56,32 @@ class SelfSwabScreen(models.Model):
     additional_symptoms = models.BooleanField()
     timestamp = models.DateTimeField(default=timezone.now)
 
+    @property
+    def hashed_msisdn(self):
+        return hash_string(self.msisdn)
+
+    def get_processed_data(self):
+        return {
+            "id": str(self.id),
+            "contact_id": self.contact_id,
+            "msisdn": self.hashed_msisdn,
+            "age": self.age,
+            "gender": self.gender,
+            "facility": self.facility,
+            "risk_type": self.risk_type,
+            "timestamp": self.timestamp.isoformat(),
+            "occupation": self.occupation,
+            "employee_number": self.employee_number,
```

Comment:
Would the employee number not be considered the same level of PII of the msisdn?

Reply:
You're right